edm4hep.yaml

```diff
@@ -222,7 +222,7 @@ datatypes:
     Description: "Event Header. Additional parameters are assumed to go into the metadata tree."
     Author: "EDM4hep authors"
     Members:
-      - int32_t eventNumber    // event number
+      - uint64_t eventNumber    // event number
       - int32_t runNumber      // run number
```

Comment:
Maybe the run number should be `uint32_t`?

Reply:
done.